enterprise_gateway/services/processproxies/crd.py

```diff
@@ -91,7 +91,8 @@ def get_container_status(self, iteration: int | None) -> str:
             if iteration:  # only log if iteration is not None (otherwise poll() is too noisy)
                 self.log.debug(
                     f"{iteration}: Waiting from CRD status from resource manager {self.object_kind.lower()} in "
-                    f"namespace '{self.kernel_namespace}'."
+                    f"namespace '{self.kernel_namespace}' Name: '{self.container_name}', "
+                    f"Status: '{application_state}', KernelID: '{self.kernel_id}'."
```

Comment:
Sorry, but this will do double logging with that from the superclass.  My suggestion was to add this only when not custom_resource status is available.